guides/source/active_record_migrations.md

```diff
@@ -1004,7 +1004,10 @@ such features, the `execute` method can be used to execute arbitrary SQL.
 Migrations and Seed Data
 ------------------------
 
-Some people use migrations to add data to the database:
+The main purpose of Rails' migration feature is to issue commands that modify the 
+schema using a consistent process. In some cases, however, migrations can also be used 
```

Comment:
I like this new paragraph :+1: 

My suggestion here is to remove "In some cases, however, " and just leave the rest of the paragraph as it is.